libs/service/service.go

```diff
@@ -30,9 +30,6 @@ type Service interface {
 	// Return true if the service is running
 	IsRunning() bool
 
-	// Quit returns a channel, which is closed once service is stopped.
-	Quit() <-chan struct{}
-
```

Comment:
Was nothing actually implementing this?

Reply:
it was just implemented in `BaseService`... most of the methods of `Service` aside from `OnStop` and `OnStart` aren't ever actually implemented by implementors.